client/layout/index.jsx

```diff
@@ -429,6 +432,7 @@ class Layout extends Component {
 						<QueryAgencies />
 					</>
 				) }
+				{ isA4AClient && <QueryClient /> }
```

Comment:
Do you think we should rename the `<QueryClient>` to `<QueryAgencyClient>` as it gives more context about itself?

Reply:
Yes, agree! I have updated it now: https://github.com/Automattic/wp-calypso/pull/91321/commits/d094a94eee32e385ac6e16b4bc56d7a9b92e0ac9